### PR TITLE
Add `IMPORT/EXPORT DATABASE`

### PR DESCRIPTION
**Pull request by hououou**

### CODE REVIEW

**Comment by hououou** [comment]
@prrao87 I do not how to add page links. Could you please help me link the  `import_export_database.md` in the correct place? ( I think it may be linked in `Cypher manual` directory.)

**Comment by prrao87** [comment]
Hi @hououou, you can link it without the `.md` extension and the relative path (just the file name). That should work.

File: src/content/docs/cypher/import_export_db.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: EXPORT/IMPORT DATABASE
+---
+
+Kùzu allows you to import/export database using query statements.
```
Comment (prrao87) [inline src/content/docs/cypher/import_export_db.md:5]: Can you add a line here at the start why you may want to import/export a database? (Something along the lines of if you need to migrate the contents of an existing database to a different version of Kùzu). Also, maybe we should mention the cases where the import/export may not work (e.g., backward compatibility is not guaranteed).

File: src/content/docs/cypher/import_export_db.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: EXPORT/IMPORT DATABASE
+---
+
+Kùzu allows you to import/export database using query statements. 
+
+## EXPORT DATABASE
```
Comment (prrao87) [inline src/content/docs/cypher/import_export_db.md:7]: Headers should be title case (Only first letter is capitalized) in this case, `## Export database`

File: src/content/docs/cypher/import_export_db.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+```
+
+The `EXPORT DATABASE` command generates three Cypher files and a series of data files in the user-defined directory.
+- `schema.cypher`: Contains the definitions of all node and edge tables.
+- `macro.cypher`: Includes the definitions of all macro functions.
+- `copy.cypher`: Contains `COPY FROM` statements for reimporting data.
+- `Data files`: These files contain the actual database data and are in CSV (by default) or Parquet format.
+
+
+## IMPORT DATABASE
```
Comment (prrao87) [inline src/content/docs/cypher/import_export_db.md:21]: Headers should be title case (Only first letter is capitalized) in this case, `## Import database`

**Review by prrao87 — CHANGES_REQUESTED** [review]

**Review by prrao87 — APPROVED** [review]
I made the required updates for clarity and added the links 👍🏽. Will merge once we make the 0.4.0 release.

**Pull request merged**